app/jobs/irs_attempt_events_batch_job.rb

```diff
@@ -0,0 +1,31 @@
+class IrsAttemptEventsBatchJob < ApplicationJob
+
+  require 'tempfile'
+
+  # copypasta
+  include JobHelpers::StaleJobHelper
+  queue_as :default
+  discard_on JobHelpers::StaleJobHelper::StaleJobError
+
+  # Get this to run at the early part of the hour
+  
+  def perform(subject_timestamp)
+    puts 'Howdy, partner'
+
+    # Probably want something more durable
+    # mktmpdir won't auto-delete if it's not called with a block:
+    # https://ruby-doc.org/stdlib-2.5.1/libdoc/tmpdir/rdoc/Dir.html
+    dir = Dir.mktmpdir(subject_timestamp.to_s)
+    file = File.new("#{dir}/#{Time.now.to_fs(:number)}", 'w')
+
+    events = IrsAttemptsApi::RedisClient.new.read_events(timestamp: subject_timestamp)
+    events.each do |event|
+      file.write event
+    end
+    file.close
+    file.path
```

Comment:
Commenting in case someone else ends up picking this up.

Right now, this reads all the events for a given hour out of Redis, writes them to a temp file, and then returns the file path (but nothing is looking at the return value, except me in the console). This is not very useful.

We want to figure out where to store this. We've discussed either S3 or Redis. With multiple servers behind a load balancer, and instances periodically recycled, we can't rely on saving the file locally.

We also want to apply encryption and gzip on this. See the `fetch_events` rake task, at least for the encryption bit.

The other bit of work on this is, once all that's working, change the endpoint to return that file, wherever it's stored, rather than generating it on the fly.